src/app/mod.rs

```
@@ -383,9 +384,10 @@ impl Surreal {
 	/// ```
 	pub async fn create(&self, resource: String, data: JsValue) -> Result<JsValue, Error> {
 		let resource = Resource::from(resource);
-		let response = match from_value::<Option<Json>>(data)? {
-			Some(data) => self.db.create(resource).content(data).await?,
-			None => self.db.create(resource).await?,
+		let data = json(&from_value::<Json>(data)?.to_string()).unwrap_or(Value::None);
```

Same here. Can we please return the error, not fall back to `Value::None`?